docs/src/modules/components/Examples/types.ts

```diff
@@ -0,0 +1,12 @@
+export interface Example {
+  title: string;
+  description: string;
+  src: string;
+  srcDark?: string;
+  href: string;
```

Comment:
Is not required
```suggestion
  href?: string;
```

Fixing this in  https://github.com/mui/toolpad/pull/4350.